src/App.java

```diff
@@ -1,5 +1,56 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
+
 public class App {
-    public static void main(String[] args) throws Exception {
-        System.out.println("Hello, World!");
+    /*loads the flashcards from flashcard.txt*/
+    private List<Flashcard> loadFlashcards(String filename) {

```

Comment:
where did you learn about `List` objects? Cite sources when using things outside the scope of the course.